slam-libraries/viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -121,12 +125,14 @@ class SLAMServiceImpl final : public SLAMService::Service {
     double rotation_weight = 1.0;
 
    private:
-    int starting_scan_number = 0;
```

Comment:
I decided to remove this for now, but we can add it back if/when we implement `input_file_pattern`.